packages/replay/src/launchdarkly.ts

```diff
@@ -0,0 +1,118 @@
+import dbg from "./debug";
+import { initialize, LDClient, LDLogger, basicLogger } from "launchdarkly-node-client-sdk";
+
+const debug = dbg("replay:launchdarkly");
+
+export type UserFeatureProfile = {
+  type: "user";
+  id: string;
+};
+export type WorkspaceFeatureProfile = {
+  type: "workspace";
+  id: string;
+};
+
+export const AnonymousFeatureProfile = {
+  type: "anonymous",
+  id: "anonymous",
+};
+
+export type FeatureProfile =
+  | typeof AnonymousFeatureProfile
+  | UserFeatureProfile
+  | WorkspaceFeatureProfile;
```

Comment:
I think it would be good to have all of those as objects or all of those as types. It's a mixed bag now and it might lead to confusion at consuming sites.